metacat-common-server/src/main/java/com/netflix/metacat/common/server/properties/ArchaiusConfigImpl.java

```diff
@@ -20,6 +20,7 @@
 //    private final DynamicStringProperty defaultTypeConverter;
 //    private final DynamicBooleanProperty isElasticSearchEnabled;
 //    private final DynamicStringProperty elasticSearchIndexName;
+    /* THIS IS USED IN TEST*/
```

Comment:
Sure. Just want to leave it there,  in case to address the dynamic properties before removing the file.